.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+# Unix-style newlines with a newline ending every file
```

Comment:
Comment is especially important if you write a "weird" or "questionable at first sight" code.